sentinel-core/transport/command/handler/version_handler.cc

```diff
@@ -0,0 +1,14 @@
+#include "sentinel-core/transport/command/handler/version_handler.h"
+
+#include <string>
+
+namespace Sentinel {
+namespace Transport {
+
+CommandResponsePtr VersionCommandHandler::Handle(
+    const CommandRequest& request) {
```

Comment:
replace `const CommandRequest& request` to `const CommandRequest&`

Reply:
OK, I'll improve this later.